fuel/schemes.py

```diff
@@ -1,10 +1,9 @@
 from abc import ABCMeta, abstractmethod
 
 import numpy
-from picklable_itertools import chain, repeat, imap, _iter
+from picklable_itertools import chain, repeat, imap, xrange, iter_
```

Comment:
Yes, now that I look at it, the `_iter` call in the code that #29 replaced was in fact entirely unnecessary. I'll ditch that one for now, but it's there if we need it (e.g. for the methods).